packages/db/src/config.ts

```diff
@@ -40,3 +47,27 @@ export const getConnectionConfigEnv = (
   password: getRequiredEnvVar('PG_PASSWORD', env),
   database: getRequiredEnvVar('PG_DATABASE', env),
 });
+
+export const getConnectionConfig = async ({
+  env,
+  secretsManager = new AWS.SecretsManager(),
+}: {
+  env: NodeJS.ProcessEnv,
+  secretsManager?: AWS.SecretsManager
+}): Promise<Knex.PgConnectionConfig> => {
+  // Using LocalStack
+  if (env.NODE_ENV === 'test') {
```

Comment:
This makes me a bit nervous to have in our deployed code rather than just passing in the env vars to make a local connection when we want to

Reply:
Is it any different than what we're doing here? https://github.com/nasa/cumulus/blob/23929edb8c27b3f8f96ae7abfbdc8485e662ab7a/packages/aws-client/src/client.ts#L44

Reply:
I'm not sure if it's a pattern that I want to double down on